### PR TITLE
Fixes to LMS Who Lives Here

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Fix missing repeating navigation if 1st instance of repeat is skipped
Duplicate who lives here for the no primary flow
Corrected routing rules

### How to review 
Check that it behaves as per slides

### CODE REVIEW

File: app/questionnaire/rules.py [hunk 1]
```diff
@@ -282,17 +282,18 @@ def evaluate_when_rules(when_rules, schema, metadata, answer_store, group_instan
     """
 
     for when_rule in when_rules:
+        group_instance_something = group_instance
```
Comment (insacuri) [inline app/questionnaire/rules.py]: group_instance_something?  The name isn't very expressive I think.

**Comment by insacuri** [comment]
I managed to break it.
- add yourself + 2 others.
- No extra students 
- yes to both sharing questions
- It then goes to an 'Who do you need to add to' page instead of relationships

**Comment by insacuri** [comment]
Same thing happens if you do just a 1 person journey (No extra people or students and any answer to the sharing questions)

File: app/questionnaire/rules.py (hunk 1 above)
Comment (jonnyshaw89) [reply app/questionnaire/rules.py]: haha, yeh I forgot about that. I was trying it to see if it fixed my issue (which it did) then forgot to call it something sensible

**Pull request closed without merging**